core/tests/unit/wasm/wasm.tests.js

```diff
@@ -109,13 +109,34 @@ describe('Unit tests for wasm Core API', function () {
   it('can process event', function () {
     // Setup
     const keyboard = loadKeyboard('k_020___deadkeys_and_backspace');
-    const state = km_core.state_create(keyboard, []);
+    const state = km_core.state_create(keyboard, []).object;
```

Comment:
Ideally we split this to assert the return value of the call as well.

Reply:
Moved to separate function.